ddtrace/contrib/internal/ray/patch.py

```diff
@@ -190,17 +192,28 @@ def traced_submit_job(wrapped, instance, args, kwargs):
     submission_id = kwargs.get("submission_id") or generate_job_id()
     kwargs["submission_id"] = submission_id
     entrypoint = kwargs.get("entrypoint", "")
-    job_name = (
-        config.service
-        or kwargs.get("metadata", {}).get("job_name", "")
-        or get_dd_job_name_from_submission_id(submission_id)
-        or get_dd_job_name_from_entrypoint(entrypoint)
-    )
+    job_name = config.service or kwargs.get("metadata", {}).get("job_name", "")
+
+    if not job_name:
+        if os.environ.get("DD_RAY_USE_ENTRYPOINT_AS_JOB_NAME", "false").lower() in ("true", "1"):
+            job_name = get_dd_job_name_from_entrypoint(entrypoint) or DEFAULT_JOB_NAME
+        else:
+            job_name = DEFAULT_JOB_NAME
 
     # Root span creation
     job_span = tracer.start_span("ray.job", service=job_name or DEFAULT_JOB_NAME, span_type=SpanTypes.RAY)
     _inject_ray_span_tags_and_metrics(job_span)
     job_span.set_tag_str(RAY_SUBMISSION_ID_TAG, submission_id)
+    if entrypoint:
+        job_span.set_tag_str(RAY_ENTRYPOINT, entrypoint)
+        entrypoint_script = get_dd_job_name_from_entrypoint(entrypoint)
+        if entrypoint_script:
+            job_span.set_tag_str(RAY_ENTRYPOINT_SCRIPT, entrypoint_script)
```

Comment:
Two point on that:
- I think it is redundant (I tested and one tag is python .telemetry-tests/playground.py and one is playground)
- I think we should be careful with sending paths without redaction which could be considered as PII or might be super long